lib/installer.js

```diff
@@ -6,7 +6,8 @@ import log from './logger';
 import crypto from 'crypto';
 
 const SE_VER = "0.17.0";
-const SE_DOWNLOAD = `http://repo1.maven.org/maven2/io/selendroid/selendroid-standalone/${SE_VER}/selendroid-standalone-${SE_VER}-with-dependencies.jar`;
+const SE_DOWNLOAD_CDNURL = process.env.APPIUM_SELENDROID_CDNURL || "http://repo1.maven.org/maven2/io/selendroid/selendroid-standalone";
```

Comment:
would be great to include this as an npm install parameter as well as an environment variable, like we do in appium-chromedriver; care to add that? otherwise this looks great.